homeassistant/components/opentherm_gw/const.py

```diff
@@ -214,324 +216,453 @@
 }
 
 SENSOR_INFO: dict[str, list] = {
-    # [device_class, unit, friendly_name, [status source, ...]]
+    # [device_class, unit, friendly_name, suggested_display_precision, [status source, ...]]
     gw_vars.DATA_CONTROL_SETPOINT: [
         SensorDeviceClass.TEMPERATURE,
         UnitOfTemperature.CELSIUS,
         "Control Setpoint {}",
+        SENSOR_FLOAT_SUGGESTED_DISPLAY_PRECISION,
         [gw_vars.BOILER, gw_vars.THERMOSTAT],
     ],
     gw_vars.DATA_MASTER_MEMBERID: [
         None,
         None,
         "Thermostat Member ID {}",
+        None,
         [gw_vars.BOILER, gw_vars.THERMOSTAT],
     ],
     gw_vars.DATA_SLAVE_MEMBERID: [
         None,
         None,
         "Boiler Member ID {}",
+        None,
         [gw_vars.BOILER, gw_vars.THERMOSTAT],
     ],
     gw_vars.DATA_SLAVE_OEM_FAULT: [
         None,
         None,
         "Boiler OEM Fault Code {}",
+        None,
         [gw_vars.BOILER, gw_vars.THERMOSTAT],
     ],
     gw_vars.DATA_COOLING_CONTROL: [
         None,
         PERCENTAGE,
         "Cooling Control Signal {}",
+        SENSOR_FLOAT_SUGGESTED_DISPLAY_PRECISION,
         [gw_vars.BOILER, gw_vars.THERMOSTAT],
     ],
     gw_vars.DATA_CONTROL_SETPOINT_2: [
         SensorDeviceClass.TEMPERATURE,
         UnitOfTemperature.CELSIUS,
         "Control Setpoint 2 {}",
+        SENSOR_FLOAT_SUGGESTED_DISPLAY_PRECISION,
         [gw_vars.BOILER, gw_vars.THERMOSTAT],
     ],
     gw_vars.DATA_ROOM_SETPOINT_OVRD: [
         SensorDeviceClass.TEMPERATURE,
         UnitOfTemperature.CELSIUS,
         "Room Setpoint Override {}",
+        SENSOR_FLOAT_SUGGESTED_DISPLAY_PRECISION,
         [gw_vars.BOILER, gw_vars.THERMOSTAT],
     ],
     gw_vars.DATA_SLAVE_MAX_RELATIVE_MOD: [
         None,
         PERCENTAGE,
         "Boiler Maximum Relative Modulation {}",
+        SENSOR_FLOAT_SUGGESTED_DISPLAY_PRECISION,
         [gw_vars.BOILER, gw_vars.THERMOSTAT],
     ],
     gw_vars.DATA_SLAVE_MAX_CAPACITY: [
         SensorDeviceClass.POWER,
         UnitOfPower.KILO_WATT,
         "Boiler Maximum Capacity {}",
+        None,
         [gw_vars.BOILER, gw_vars.THERMOSTAT],
     ],
     gw_vars.DATA_SLAVE_MIN_MOD_LEVEL: [
         None,
         PERCENTAGE,
         "Boiler Minimum Modulation Level {}",
+        None,
```

Comment:
Screenshot of snippet from `pyotgw`:
![afbeelding](https://github.com/home-assistant/core/assets/2793431/66575463-ab26-47cd-baf5-56721269234d)

The `DATA_SLAVE_MIN_MOD_LEVEL` is a `u8` by OpenTherm standards. We could add a digit of precision but by definition it will never be used.